benefits/core/migrations/0002_sample_data.py

```diff
@@ -149,6 +150,7 @@ def load_sample_data(app, *args, **kwargs):
         unverified_title=_("eligibility.pages.unverified.mst_cc.title"),
         unverified_headline=_("eligibility.pages.unverified.mst_cc.headline"),
         unverified_blurb=_("eligibility.pages.unverified.mst_cc.p[0]"),
+        eligibility_confirmed_headline=("enrollment.pages.index.mst_login.eligibility_confirmed.headline"),
```

Comment:
```suggestion
        eligibility_confirmed_headline=("enrollment.pages.index.mst_cc.eligibility_confirmed.headline"),
```

Reply:
Should be addressed in https://github.com/cal-itp/benefits/pull/1093/commits/188944a1f4a9013a7ab8abaa61ea11c6a027f598

Reply:
There's a further issue with this, which is contributing to the `bin/makemessages.sh` problem -- the gettext helper function `_` needs to be called here, e.g. 


```python
eligibility_confirmed_headline=_("enrollment.pages.index.mst_cc.eligibility_confirmed.headline"),
```